webapp/src/client/client.js

```diff
@@ -95,4 +97,20 @@ export default class Client {
             body,
         });
     };
+
+    clientExecuteCommand = async (getState, command) => {
```

Comment:
Passing redux state and using selectors in the client is a bit weird. I think you should add an action in `actions/index.js` `executeCommand` that you can dispatch (from within the `restartInstallation` action) with the command to execute, and have that action perform the state selecting (`getCurrentChannel`, `getCurrentTeamId`, etc) before passing all 3 values (current channel, current team, command) to this client function

Reply:
Right, good point. I saw this type of structure used in another plugin way back and never questioned it, but I see that it's a bit strange.

I think the new commit is what you asked for. Lemme know if I missed something.